Error  iterators/generators require regenerator-runtime, which is too heavyweight

C:\Users\lukas\workspace\snake_js\index.js
   6:3   error  iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations
  no-restricted-syntax

--- a/index.js
+++ b/index.js
@@ -3,8 +3,9 @@ import Snake from "./src/Snake.js";
 const snake = new Snake(10, 10);
 for (;;) {
   snake.boardGame.clear();
-  for (const [x, y] of snake) {
-    snake.boardGame.drawPixel([x, y, "x"]);
+  const snakeToDraw = snake.getFullSnake();
+  for (let i = 0; i < snakeToDraw.length; i += 1) {
+    snake.boardGame.drawPixel([...snakeToDraw[i], "x"]);
   }
   snake.boardGame.drawPixel([...snake.food, "T"]);
   console.clear();

--- a/src/Snake.js
+++ b/src/Snake.js
@@ -18,17 +18,17 @@ export default class Snake {
     this.length = [this.tail, ...this.body, this.head].length;
   }
 
-  #getFullSnake = () => {
+  getFullSnake = () => {
     const fullSanke = [this.tail, ...this.body, this.head];
     return fullSanke;
   };
 
   toString() {
-    return JSON.stringify(this.#getFullSnake());
+    return JSON.stringify(this.getFullSnake());
   }
 
   [Symbol.iterator]() {
-    return this.#getFullSnake()[Symbol.iterator]();
+    return this.getFullSnake()[Symbol.iterator]();
   }
 
   move(nextSpot) {
@@ -70,7 +70,7 @@ export default class Snake {
   }
 
   preventModificationSnake(nextSpot) {
-    return this.#getFullSnake().some((element) =>
+    return this.getFullSnake().some((element) =>
       this.#equals(element, nextSpot)
     );
   }

